lib/Result.cpp

```diff
@@ -0,0 +1,67 @@
+
+#include "Result.h"
+
+#include "llvm/IR/DebugInfo.h"
+#include "llvm/IR/DebugInfoMetadata.h"
+#include "llvm/IR/Function.h"
+
+#include <string>
+
+using namespace Mutang;
+using namespace llvm;
+
+std::string Result::calculateCallerPath(MutationResult *mutationResult) {
+  Testee *currentTestee = mutationResult->getTestee();
+
+  char formatBuffer[512];
+
+  std::vector<std::string> strings;
+
+  /// Currently we don't print functions, only "file:line". Later we will and
+  /// for that we will have to demangle them.
+
+  /// Last path component: mutation point itself.
+  auto mutationPoint = mutationResult->getMutationPoint();
+  Instruction *instruction = dyn_cast<Instruction>(mutationPoint->getOriginalValue());
+  std::string fileName = instruction->getDebugLoc()->getFilename();
+  std::string line = std::to_string(instruction->getDebugLoc()->getLine());
+
+  ///Function *function = instruction->getFunction();
+  ///assert(function && "Expected function");
+  ///std::string functionName = function->getName().str();
+
+  snprintf(formatBuffer, sizeof(formatBuffer), "-- %s:%s\n", fileName.c_str(), line.c_str());
```

Comment:
I'd make this function to return the path as an array of strings, and then format accordingly inside of reporter. This way we may save it in different formats (e.g. in pre-formatted and in csv form), and experiment which one works better.